crates/ide-assists/src/handlers/generate_delegate_trait.rs

```diff
@@ -1589,12 +1600,12 @@ where
     b : B<T>,
 }
 
-impl<T, T0> Trait<T> for S<T0>
+impl<T, T2> Trait<T> for S<T2>
 where
-    T0: AnotherTrait
+    T2: AnotherTrait
 {
     fn f(&self, a: T) -> T {
-        <B<T0> as Trait<T>>::f(&self.b, a)
+        <B<T2> as Trait<T>>::f(&self.b, a)
     }
```

Comment:
These should be `T1` not `T2` no? 🤔

Reply:
Perhaps I should refactor the logic of renaming in `generate_delegate_trait`. 🙊